x/stakeibc/keeper/community_pool_test.go

```diff
@@ -166,3 +169,116 @@ func (s *KeeperTestSuite) TestQueryCommunityPoolBalance_FailedQuerySubmission()
 	err := s.App.StakeibcKeeper.QueryCommunityPoolBalance(s.Ctx, invalidHostZone, icaAccountType, Atom)
 	s.Require().ErrorContains(err, "Error submitting query for pool ica balance")
 }
+
+
+
+type RedeemCommunityPoolTokensTestCase struct {
+	hostZone                     types.HostZone
+	initialDummyTokens           sdkmath.Int
+	communityPoolHoldingAddress  sdk.AccAddress
+}
+
+func (s *KeeperTestSuite) SetupRedeemCommunityPoolTokens() RedeemCommunityPoolTokensTestCase {
+	s.CreateTransferChannel(HostChainId)
+
+	// Create relevant module and ICA accounts
+	depositAddress := s.TestAccs[0]
+	communityPoolHoldingAddress := s.TestAccs[1]
+	communityPoolReturnICAAddress := DelegationICAAddress // need an address on HostChain (starts cosmos)
```

Comment:
wdyt about renaming this to `HostICAAddress` just so we can reuse it? I think it's only used in one other place

Reply:
Sure that would be more general.  I was surprised in the createICAChannel testing function, when it registers and ICA, it seems to do it on the stride side instead of making an array of HostSide ICAs.  It would be easy to modify that function to do both (so add that it makes ICAs on the hostSide) and then we would have more than one potential address to test with (I can imagine we want to test sending ICA to ICA at somepoint)

Reply:
The tricky thing is that the host side is also technically a "stride chain" so it will still have a stride prefix.

I agree it would be nice to actually test the interaction of the two chains more in the unit tests. I think when we built this framework, it was a bit complex to setup but probably something we should revisit soon! But I think for now we can just scope the tests to the point at which the message is sent